src/client/lazy-app/Compress/Output/custom-els/TwoUp/index.ts

```diff
@@ -84,20 +77,36 @@ export default class TwoUp extends HTMLElement {
         }</svg>
       `}</div>`;
 
+    // Watch for element size changes.
+    if ('ResizeObserver' in window) {
+      this._resizeObserver = new ResizeObserver(() => this._resetPosition());
+      this._resizeObserver.observe(this);
+    } else {
+      window.addEventListener('resize', this._onResize);
+    }
+
+    window.addEventListener('keydown', this._onKeyDown);
+
     if (!this._everConnected) {
       this._resetPosition();
       this._everConnected = true;
     }
   }
 
+  disconnectedCallback() {
+    window.removeEventListener('keydown', this._onKeyDown);
+    window.removeEventListener('resize', this._onResize);
```

Comment:
We should have caught these at review time 😄. I guess we would have if it was a Preact component, but we're less experienced with web components.